pkg/local_object_storage/metabase/delete.go

```diff
@@ -89,7 +90,7 @@ func (db *DB) delete(tx *bbolt.Tx, addr oid.Address, refCounter referenceCounter
 	}
 
 	// unmarshal object, work only with physically stored (raw == true) objects
-	obj, err := db.get(tx, addr, false, true)
+	obj, err := db.get(tx, addr, false, true, currEpoch)
```

Comment:
How do we ever delete expired object from the metabase?

Reply:
third arg (`false` here) controls checking object status. if an object is presented, it will be deleted, despite its status